tests/test-a101-pins.js

```diff
@@ -2,10 +2,10 @@
 
 // Test Arduino101Pins API
 
-var pins = require('arduino101_pins');
-var gpio = require('gpio');
-var pwm = require('pwm');
-var aio = require('aio');
+var pins = require("arduino101_pins");
+var gpio = require("gpio");
+var pwm = require("pwm");
+var aio = require("aio");
 var assert = require("Assert.js");
```

Comment:
Actually we're trying to move to all single quotes instead of double quotes, that's what Google JS Style Guide recommends.